water.md

```diff
@@ -20,59 +20,172 @@ Water demand representations have been constructed for six major sectors: agricu
+
+When renewable water supplies are abundant relative to demand, water prices remain low (or negligible), and non-renewable groundwater and desalinated water do not contribute to supply. In this case, water supplies are not binding and do not constrain the production of goods and services. When water demands approach or exceed accessible renewable supplies, water prices rise more rapidly such that more expensive non-renewable groundwater and desalinated water contribute to supply. Generally, renewable water supplies are lower in cost and are utilized first and to a greater extent before depending on more expensive non-renewable groundwater or desalinated water. Over time, basins may produce from all three sources of water depending on the market price of water and the cost of alternative water resources in each basin.
+
+In order to improve the solution behavior of all GCAM markets with interlinked water dependencies, and to limit the total number of new markets to solve, renewable and non-renewable water resources have been combined into one effective resource supply curve. This limits the number of new markets to solve to 235, the number of water basins included. Renewable and non-renewable water resources are separately tracked, however. In GCAM, two Sub-Resource objects are included within the Resource object for each water basin, one each to represent renewable and non-renewable water resources.  The Sub-Resource supply curves are additive and together represent the total supply of water.
+
+Adding renewable and non-renewable resource curves into one is conceptually a new feature to GCAM. Discontinuity in the aggregate supply curve due to changing renewable potential and/or depletion of nonrenewable water resources has been accounted for by ensuring that the two Sub-Resource curves are overlapping so as to ensure continuity of the aggregate supply curve.
+The ability to combine and overlay multiple supply curves into one provides great flexibility in the potential for alternative characterization of water resources and their contributions to the overall supply. For instance, both renewable and non-renewable resources can be further disaggregated, such as by separating renewable water into surface and ground water (recharge) portions or separating fossil groundwater by alternative aquifer characteristics. The shape of the multiple resource curves and the degree to which they overlap or not can determine water use behavior, such as the simultaneous or sequential production of alternative water resources.
+
+One notable feature of the GCAM's Renewable Sub-Resource object is the ability to vary the maximum available resource potential by time period. This feature enables the assessment of climate impacts on water availability. Projected future changes in precipitation from climate models (GCMs) are utilized to calculate river runoff and changes in the maximum available potential of renewable water supplies by time and basin.

```

Comment:
This section and the next are generally more detailed description than we typically include in this form of documentation.  Which was by design.  That being said it leaves to really publicly available space for this level of documentation.  So I could be persuaded to leave it.

Reply:
The purpose of the more detailed information serves to let the general audience know that we have thought about water supply issues in more detail and that we have the capability for addressing them. For the max resource, that info was important so let users know that we have a way to incorporate climate change impacts. I'd like to keep the more detailed documentation.